frontend/packages/dev-console/src/components/pipelineruns/logs/LogsWrapperComponent.tsx

```diff
@@ -14,7 +14,7 @@ type LogsWrapperComponentProps = {
 const LogsWrapperComponent: React.FC<LogsWrapperComponentProps> = ({ obj, ...props }) => {
   const ref = React.useRef(obj?.data);
   if (!_.isEmpty(obj?.data)) {
-    ref.current = obj.data;
+    ref.current = _.get(obj, ['data', 0]);
```

Comment:
```suggestion
    ref.current = obj.data[0];
```

Reply:
this is throwing lint error for array destructuring

Reply:
you can use 
```suggestion
    ref.current = obj.data?.[0];
```
which should not give lint error

Reply:
updated